### PR TITLE
Learn bias

### PR DESCRIPTION
**Pull request by mstrazar**
Added learning of bias to BRISMF and SVD++. Objective function values are slightly lower due to extra parameters.

### CODE REVIEW

File: orangecontrib/recommendation/rating/brismf.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def _compute_objective(users, items, global_avg, dUsers, dItems, P, Q, target,
     return objective.sum()
 
 
-def _matrix_factorization(data, bias, shape, order, K, steps, alpha, beta,
+def _matrix_factorization(data, bias, shape, order, K, steps, alpha, beta, alpha_bias=0,
```
Comment (kernc) [inline orangecontrib/recommendation/rating/brismf.py]: Update docstring.

File: orangecontrib/recommendation/rating/svdplusplus.py [hunk 2]
```diff
@@ -166,12 +176,13 @@ class SVDPlusPlusLearner(Learner):
 
     name = 'SVD++'
 
-    def __init__(self, K=5, steps=25, alpha=0.07, beta=0.1, min_rating=None,
+    def __init__(self, K=5, steps=25, alpha=0.07, alpha_bias=0.007, beta=0.1, min_rating=None,
```
Comment (kernc) [inline orangecontrib/recommendation/rating/svdplusplus.py]: Update docstring here in particular.

File: orangecontrib/recommendation/rating/brismf.py [hunk 3]
```diff
@@ -143,10 +152,11 @@ class BRISMFLearner(Learner):
 
     def __init__(self, K=5, steps=25, alpha=0.07, beta=0.1, min_rating=None,
                  max_rating=None, preprocessors=None, verbose=False,
-                 random_state=None):
+                 random_state=None, alpha_bias=0.007):
```
Comment (kernc) [inline orangecontrib/recommendation/rating/brismf.py]: And here.

File: orangecontrib/recommendation/rating/brismf.py (hunk 1 above)
Comment (mstrazar) [inline orangecontrib/recommendation/rating/brismf.py]: Updated docstring. Salva will update other methods (currently empty) when implementation is final.

**Comment by salvacarrion** [comment]
I had too many conflicts that I couldn't merge it, so I've added them manually. Basically, when the pull request was done I was in the middle of a major normalization of arguments names, methods and classes.

**Pull request closed without merging**